### PR TITLE
CLOUDP-164972: Migrate Atlas CLI "Projects" Client to v2 SDK Client

### PR DESCRIPTION
**Pull request by swattyT**
## Proposed changes

_Jira ticket:_ [CLOUDP-164972](https://jira.mongodb.org/browse/CLOUDP-164972)

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by swattyT — COMMENTED** [review]
EVG Patch-https://spruce.mongodb.com/version/6412fc700305b99b64259949/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

File: internal/cli/iam/projects/create.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	atlasCreateTemplate = "Project '{{.ID}}' created.\n"
+	atlasCreateTemplate = "Project '{{.Id}}' created.\n"
```
[the remark this answers is not in the document]
Comment (gssbzn) [reply internal/cli/iam/projects/create.go:33]: you can conditionally change the template, for example
https://github.com/mongodb/mongodb-atlas-cli/blob/0230e778bacf173024fd83577b8a1e26b5ce9438/internal/cli/iam/projects/create.go#L159-L161

[the remark this answers is not in the document]
Comment (wtrocki) [reply internal/cli/iam/projects/create.go:33]: Or duplicate template. Please consider follow-up plans so we do not generate unnecessary work on your side. We should look for ways to minimize our impact and work on this.

**Comment by swattyT** [comment]
Closing Investigative PR.

**Pull request closed without merging**